views/job.html

```diff
@@ -2,7 +2,7 @@
 	<h2>
 	  <a href="http://github.com/{{username}}/{{name}}">{{name}}</a>
 	</h2>
-	<h3>{{#language}}{{language}} - {{/language}}Creator &amp; Owner</h3>
+	<h3>{{#language}}{{language}} &ndash; {{/language}}Creator &amp; Owner</h3>
```

Comment:
Here should be em-dash, right? 

http://english.stackexchange.com/questions/2116/when-should-i-use-an-em-dash-an-en-dash-and-a-hyphen

> An em-dash is typically used as a stand-in for a comma or parenthesis to separate out phrases—or even just a word—in a sentence for various reasons (i.e. an appositive).


> An en-dash is used to connect values in a range or that are related. A good rule is to use it when you're expressing a "to" relationship.